chia/server/server.py

```diff
@@ -467,11 +466,17 @@ async def start_client(
                 self.log.info(f"Connected to a node with the same peer ID, disconnecting: {target_node} {peer_id}")
                 return False
 
+            server_port: uint16
+            try:
+                server_port = self.get_port()
+            except ValueError:
+                server_port = uint16(0)
```

Comment:
thinking about this. I guess in the code today, I will perform the handshake and say I'm on port XYZ, but I may not be actually listening at all on the port, or using it in at all (is listen is false when the server started).